dataland-frontend/src/components/forms/parts/UploadReports.vue

```diff
@@ -0,0 +1,288 @@
+<template>
+  <div v-if="!editMode" class="col-3 p-3 topicLabel">
+    <h4 id="uploadReports" class="anchor title">Upload company reports</h4>
+    <p>Please upload all relevant reports for this dataset in the PDF format.</p>
+  </div>
+  <!-- Select company reports -->
+  <div class="col-9 formFields uploaded-files">
+    <h3 class="mt-0">Select company reports</h3>
+    <FileUpload
+      name="fileUpload"
+      ref="fileUpload"
+      accept=".pdf"
+      @select="onSelectedFiles"
+      :multiple="true"
+      :maxFileSize="maxFileSize"
+      invalidFileSizeMessage="{0}: Invalid file size, file size should be smaller than {1}."
+      :auto="false"
+    >
+      <template #header="{ chooseCallback }">
+        <div class="flex flex-wrap justify-content-between align-items-center flex-1 gap-2">
+          <div class="flex gap-2">
+            <PrimeButton
+              data-test="upload-files-button"
+              @click="chooseCallback()"
+              icon="pi pi-upload"
+              :label="editMode ? 'ADD REPORTS' : 'SELECT REPORTS'"
+            />
+          </div>
+        </div>
+      </template>
+      <template #content="{ files, removeFileCallback }">
+        <div v-if="files.length > 0" data-test="uploaded-files">
+          <div
+            v-for="(file, index) of files"
+            :key="file.name + index"
+            class="flex w-full align-items-center file-upload-item"
+          >
+            <span data-test="uploaded-files-title" class="font-semibold flex-1">{{ file.name }}</span>
+            <div data-test="uploaded-files-size" class="mx-2 text-black-alpha-50">
+              {{ formatBytesUserFriendly(file.size, 3) }}
+            </div>
+            <PrimeButton
+              data-test="uploaded-files-remove"
+              icon="pi pi-times"
+              @click="removeReportFromFilesToUpload(file, removeFileCallback, index)"
```

Comment:
`file` hast type `File`, but the first argument of `removeReportFromFilesToUpload` accepts a type `Record<string,string>`.

Obviously it still works. Do you know why?

And could we correct this?